roles/neutron-control/tasks/monitoring.yml

```diff
@@ -22,3 +22,7 @@
                        args='-S neutron --scheme {{ monitoring.graphite.host_prefix }}'
   notify: restart sensu-client
 
+- name: lbaas sla metrics
+  sensu_metrics_check: name=lbaas-sla-metrics plugin=metrics-os-api.py
+                       args='-S lbaas --scheme {{ monitoring.graphite.host_prefix }}'
+  notify: restart sensu-client
```

Comment:
Shouldn't this be contingent on lbaas being enabled? I'm changing some of the logic in #1366 and I think you can use that style here in this PR for this check too.